terraform/00-init.tf

```diff
@@ -0,0 +1,29 @@
+terraform {
+  required_version = ">= 0.12.25"
+  backend "azurerm" {}
+
+  required_providers {
+    azurerm = "~> 2.31.1"
+    sendgrid = {
+      source  = "Trois-Six/sendgrid"
+      version = "0.1.6"
+    }
+  }
+}
+
+provider "azurerm" {
+  version = "2.31.1"
```

Comment:
you don't need to specify the version if defined in terraform config
```suggestion
```

Reply:
removed